src/locales/en/translation.json

```diff
@@ -2,12 +2,47 @@
   "paymentProcessing": {
     "amount": {
       "header": "100% of your support will go to the merchant toward their operating costs. Your transaction is secure, encrypted and directly goes to the owner for sustaining their business.",
+      "light_up_chinatown": "100% of donation goes towards permanent light fixtures throughout Chinatown.",
       "label1": "Select amount",
       "label2": "Other:",
       "minimum": "Minimum",
       "maximum": "Maximum",
       "amount": "amount",
-      "submit": "Next"
+      "submit": "Next",
+      "adopt_lantern_tier_2": "Adopt-a-Lantern!",
+      "adopt_lantern_tier_3": "Adopt-a-Lantern, Keep-a-Lantern!",
+      "light_up_chinatown_address": {
+        "disclaimer_1": "You will be able to pick up your custom lantern at ",
+        "disclaimer_2": "This information will also be emailed to you.",
+        "address": "Chinese Chamber of Commerce, 33 Bowery # C202, New York, NY 10002.",
+        "address_button": " I’m from out of town and would like my lantern shipped to me.",
+        "cancel_address_button": "Cancel adding address"
+      },
+      "tier_2_donation": "Thank you for donating $45 or more.",
+      "tier_3_donation": "Thank you for donating $150 or more.",
```

Comment:
Can we have this use the passed in consts? Making sure in case we change those they reflect here too

Reply:
Can we also rename them to tier_1 and tier_2 respectively from tier_2 and tier_3 to be consistent

Reply:
the wording is a little weird. Technically it should be be tier_2 and tier_3 because tier 1 is just anything lower than a $45 donation. For now it makes more sense to do tier_2_min and tier_3_min instead of tier_2_limit and tier_3_limit. @jacoblb64  @danthaman44  @justintmckibben  @aliceluo2048  what do you guys think? keep change tier_2_limit -> tier_2_min or tier_2_donation to tier_1_donation.

Reply:
What about

`TIER_2_MIN = 45` and 'tier_2_donation = "Thank you for donating $45 or more."`

`TIER_3_MIN = 150` and 'tier_3_donation = "Thank you for donating $150 or more."`

Conceptually it would also be

`TIER_1_MIN = 0` and `tier_1_donation = "Thank you for donating."`